account_financial_report/reports.xml

```diff
@@ -5,112 +5,122 @@
 
     <!-- General Ledger -->
     <report
-      id="action_report_general_ledger_qweb"
-      model="report_general_ledger"
+      id="action_print_report_general_ledger_qweb"
+      model="general.ledger.report.wizard"
       string="General Ledger"
       report_type="qweb-pdf"
-      name="account_financial_report.report_general_ledger_qweb"
-      file="account_financial_report.report_general_ledger_qweb"
+      menu="False"
+      name="account_financial_report.general_ledger"
+      file="account_financial_report.general_ledger"
       />
     <report
-      id="action_report_general_ledger_html"
-      model="report_general_ledger"
+      id="action_print_report_general_ledger_html"
+      model="general.ledger.report.wizard"
       string="General Ledger"
       report_type="qweb-html"
-      name="account_financial_report.report_general_ledger_qweb"
-      file="account_financial_report.report_general_ledger_html"
+      menu="False"
+      name="account_financial_report.general_ledger"
```

Comment:
**The `name` attribute can not be the same between report records.**

When you print a PDF report, a JS action is triggered that later calls back to Odoo with only report_name as parameter to the controller. Reference here:
https://github.com/odoo/odoo/blob/85fe44a7298ef9883160359814e74e39b7e10873/addons/web/controllers/main.py#L1588-L1600

So if `_get_report_from_name` in that piece of code gets the "html" report record instead (which doesn't have paperformat set), a default paperformat is used.

This has already been reported on #558 yet wasn't fixed.

Reply:
Can you send a PR to this branch?

Reply:
Do you mean on the forked repo?
I probably could, but not this week, I guess.
Simply adding `_html` to the HTML report name would work and be quick to do, but "Coder OCD :smile:" doesn't allow me to leave the PDF one without `_pdf` at the end, which then requires some more changes elsewhere.